### PR TITLE
Add anchors for linking purposes

### PR DESCRIPTION
**Pull request by tlblessing**
## Description

Added anchors so that docs can link to sections.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

### CODE REVIEW

File: docs/src/main/sphinx/connector/hive-s3.rst [hunk 1]
```diff
@@ -10,6 +10,8 @@ uses an S3 prefix, rather than an HDFS prefix.
 Trino uses its own S3 filesystem for the URI prefixes
 ``s3://``, ``s3n://`` and  ``s3a://``.
 
+.. _hive-s3-config-props:
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/hive-s3.rst:13]:
```suggestion
.. _hive-s3-configuration:
```

File: docs/src/main/sphinx/connector/hive-s3.rst [hunk 2]
```diff
@@ -284,6 +286,8 @@ Property name                                           Description
                                                         value that is not used in any of your IAM ARNs.
 ======================================================= =================================================================
 
+.. _hive-s3-tuning-props:
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/hive-s3.rst:289]:
```suggestion
.. _hive-s3-tuning-configuration:
```

**Review by ebyhr — APPROVED** [review]

**Review by mosabua — APPROVED** [review]
I agree with the suggestions from @ebyhr .. then good to go.

**Comment by tlblessing** [comment]
Thank you @ebyhr and @mosabua

**Pull request merged**